e2e/tests/cohorts/cohorts-create.spec.ts

```diff
@@ -11,7 +11,7 @@ import {findOrCreateWorkspace, signIn} from 'utils/test-utils';
 import {waitForText, waitWhileLoading} from 'utils/waits-utils';
 
 
-describe('User can create new Cohorts', () => {
+describe.skip('User can create new Cohorts', () => {
```

Comment:
skipped most tests for testing quicker. Changes will be reverted back before merge.